TrackletAlgorithm/ProjectionCalculator.h

```diff
@@ -80,12 +78,13 @@ template<
     TrackletProjectionMemory<BARREL2S> projout_barrel_2s[TP::N_PROJOUT_BARREL2S],
     TrackletProjectionMemory<DISK> projout_disk[TP::N_PROJOUT_DISK])
 {
-
+#pragma HLS latency min=13 max=13
+  
   if (!valid) return;
 
-  int ipage =  trackletIndex >> 7;
+  int ipage =  trackletIndex >> 7; //FIXME hardcoded values
```

Comment:
@aryd Please replace the magic 7 with `kNBits_MemAddr`:
```suggestion
  int ipage =  trackletIndex >> kNBits_MemAddr;
```